tests/unit/test_query_language.py

```diff
@@ -63,3 +67,45 @@ def test_dataset() -> None:
             rhs=Value(2000),
         ),
     }
+
+
+# The problem: We'd like to test that operations on query language (QL) elements return
+# the correct query model (QM) elements. We like tests that emphasise what is being
+# tested, and de-emphasise the scaffolding. We dislike test code that looks like
+# production code.
+@pytest.fixture
+def select_patient_table():
+    return SelectPatientTable(name="patient_table")
+
+
+@pytest.fixture
+def select_column(select_patient_table):
+    return SelectColumn(select_patient_table, name="column")
+
+
+qm = operator.attrgetter("qm_node")
+
+
+def assert_produces(ql_element, qm_element):
+    assert qm(ql_element) == qm_element
+
+
+class TestIntSeries:
+    def test_le_value(self, select_column):
+        assert_produces(
+            IntSeries(select_column) <= 2000, Function.LE(select_column, Value(2000))
+        )
```

Comment:
You've solved the problem of having to call `.qm_node` in the tests nicely by explicitly constructing the QL object here. :+1: